.gitmodules

```diff
@@ -0,0 +1,5 @@
+[submodule "rta_makedata"]
+	path = rta-makedata
+	url = https://github.com/arangodb/rta_makedata
+[submodule "rta-makedata/"]
```

Comment:
Why do we have 2 submodules here? Can we leave "rta-makedata" only?

Reply:
oops, moving the repo wasn't so easy after all. fixed.